.github/workflows/build-java17.yml

```diff
@@ -0,0 +1,41 @@
+name: "Build Java 17"
+
+on:
+  push:
+    branches:
+    - georchestra-gn4.4.x
+  pull_request:
+  workflow_dispatch:
+
+env:
+  DOCKER_TAG: 4.4.x
+
+jobs:
+  build:
+    if: "!startsWith(github.event.head_commit.message, '[skip ci] ')"
+    runs-on: ubuntu-latest
+    timeout-minutes: 40
+    steps:
+    - name: "Checking out"
+      uses: actions/checkout@v2.4.0
+      with:
+        submodules: 'recursive'
+
+    - name: "Setting up Java"
+      uses: actions/setup-java@v2
+      with:
+        java-version: '17'
+        distribution: 'adopt'
+
+    - name: "Installing GeoNetwork"
+      run: mvn clean install -DskipTests
+
+    - name: "run tests"
+      run: mvn verify -Pit
```

Comment:
consider also the following comment:
https://github.com/georchestra/georchestra/issues/4510#issuecomment-2959881639